src/app/containers/ErrorMain/index.jsx

```diff
@@ -16,6 +16,10 @@ const ErrorMetadata = ({ locale, messaging, brandName }) => {
       <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
       <meta charSet="utf-8" />
       <meta name="robots" content="noindex,nofollow" />
+      <meta
+        name="viewport"
+        content="width=device-width, initial-scale=1, minimum-scale=1"
+      />
       <title>
         {title} - {brandName}
```

Comment:
Feels like `<meta name="theme-color"/>` should also be something that is pretty much universal -- & given we have the brand name, is there any reason why service colour couldn't also be abstracted into the service provider?

Reply:
Done :)